### PR TITLE
simplify build

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

File: packaging/nuget/nservicebus.azurestoragequeues.nuspec [hunk 1]
```diff
@@ -20,6 +20,6 @@
     </dependencies>
   </metadata>
   <files>
-    <file src="..\..\binaries\NServiceBus.Azure.Transports.WindowsAzureStorageQueues.???" target="lib\net452" />
+    <file src="..\..\binaries\net452\NServiceBus.Azure.Transports.WindowsAzureStorageQueues.???" target="lib\net452" />
```
Comment (Scooletz) [inline packaging/nuget/nservicebus.azurestoragequeues.nuspec]: is the directory `net452` auto generated now? Is it a feature of a new build system? @SimonCropp

**Comment by Scooletz** [comment]
Is it ready for a review @SimonCropp ?

File: packaging/nuget/nservicebus.azurestoragequeues.nuspec (hunk 1 above)
Comment (SimonCropp) [inline packaging/nuget/nservicebus.azurestoragequeues.nuspec]: Yeah it means each target framework gets its own output directory

**Comment by SimonCropp** [comment]
Yep good for review

**Pull request merged**